avogadro/quantumio/gaussianfchk.cpp

```diff
@@ -129,58 +129,60 @@ void GaussianFchk::processLine(std::istream& in)
   } else if (Core::contains(key, "UHF")) {
     m_scftype = Uhf;
   } else if (key == "Number of atoms" && list.size() > 1) {
-    m_numAtoms = Core::lexicalCast<int>(list[1]);
+    m_numAtoms = Core::lexicalCast<int>(list[1]).value_or(0);
   } else if (key == "Charge" && list.size() > 1) {
-    m_charge = Core::lexicalCast<int>(list[1]);
+    m_charge = Core::lexicalCast<int>(list[1]).value_or(0);
   } else if (key == "Multiplicity" && list.size() > 1) {
-    m_spin = Core::lexicalCast<int>(list[1]);
+    m_spin = Core::lexicalCast<int>(list[1]).value_or(0);
```

Comment:
I think we'd want the default spin to be 1 here. Spin multiplicity of 0 would likely be problematic for other parts of the code.

Reply:
Thank you for review. Fixed.